cpp/inc/bond/ext/grpc/detail/io_manager_tag.h

```diff
@@ -15,13 +15,18 @@ namespace bond { namespace ext { namespace gRPC { namespace detail {
     /// operation in its implementation of \ref invoke.
     struct io_manager_tag
     {
-        virtual ~io_manager_tag() { }
+        virtual ~io_manager_tag() = default;
 
         /// @brief Called when this instance is dequeued from a completion
         /// queue.
         ///
         /// @param ok whether or not the initial operation succeeded
         virtual void invoke(bool ok) = 0;
+
+        io_manager_tag* tag()
```

Comment:
Minor: add a brief doc comment? Since this is an interface--albeit an internal one--it's been documented to outline the contract it expects.

```cpp
/// @return Returns a %tag value suitable for passing to completion queue routines.
```